api/routes/news.js

```diff
@@ -2,87 +2,217 @@ const express = require('express');
 const app = express();
 app.use(express.json());
 const CustomError = require('../../error-handler/index')
-const validateNews = require('../../validators/validate')
-const newsArray = [
-    {
-        id: 1,
-        name: "Nitin"
-    },
-    {
-        id: 2,
-        name: "Slava"
-    },
-    {
-        id: 3,
-        name: "Jack"
-    },
-    {
-        id: 4,
-        name: "Kohli"
-    }
-];
+const mongoose = require('mongoose')
+const News = require('../models/news')
+const bcrypt = require('bcrypt')
+const User = require('../models/user')
+const passport = require('passport')
+const facebookStrategy = require('passport-facebook')
+
+app.get('/users/login', (req,res) => {    
+    res.render('login')
+});
+
+app.get('/users/register', (req,res) => {    
+    res.render('register')
+});
 
 app.get('/news', (req, res, next) => {
-    res.status(200).json({
-        newsArrayJSON: newsArray
-    })    
+    News.find()
+    .select('name _id')
+    .exec()
+    .then(data => {
+        if(!data) {
+            throw new CustomError('Internal Server Error', 500)
+        }
+        if(data.length >= 0) {
+            res.status(200).json({
+                count: data.length,
+                newsArray: data                
+            })
+        } else {
+            setTimeout(function () {
```

Comment:
Looks like you missed that place to update :)

Reply:
Updated